CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Unreleased
+## [0.23.1] - 2023-04-20
```

Comment:
Same question as Android

Reply:
Generated from MCM

Reply:
The date should be the day of the actual release